src/udev/hidraw_id/hidraw_id.c

```diff
@@ -0,0 +1,114 @@
+/* SPDX-License-Identifier: GPL-2.0+ */
+/*
+ * Modified from "u2f_hidraw_id.c" on 2019-08-19
+ *
+ * Original file header:
+ *
+ * Copyright (c) 2014-2015 Andrew Lutomirski
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Since I based this on an existing GPL project, I decided to keep the modification information in. Let me know if this can also go.

Reply:
our code is generally LGPL2, not GPL. it's OK to copy in GPL code if it's at the fringes, like it is here. But I think it would be a good idea to add an explicit line that clarifies why this file is GPL and not LGPL.

Reply:
At the current state, this commit doesn't really share a line with the original code anymore. Maybe this would even warrant removing the "modified" header and licensing under LGPL?

Reply:
I went ahead and changed the license to LGPL. I hope @amluto approves, but I can always revert this change if it should not be appropriate.

Reply:
Maybe add a comment "Inspired by …" to be friendly